content/gettingstarted/index.adoc

```diff
@@ -159,3 +159,27 @@ external_services:
   ...
 ----
 {nbsp} +
+
+=== Reducing Permissions in OpenShift
+
+By default Kiali will run with the cluster wide `kiali` role.
+
+If you do not want to run Kiali with cluster wide permissions its possible to reduce these permissions to individual namespaces.
```

Comment:
I suggest: "If you prefer not to run Kiali with cluster wide permissions, it is possible to reduce these permissions to individual namespaces."

Reply:
Updated.